packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -110,7 +110,7 @@ export function useBlockProps( props = {}, { __unstableIsHtml } = {} ) {
 	return {
 		...wrapperProps,
 		...props,
-		ref,
+		ref: useMergeRefs( [ ref, useEventHandlers( clientId ) ] ),
```

Comment:
Nit: I have a preference for hooks to be used before the "return" to avoid future issues due to potential early returns added later.

Reply:
Ok, I don't mind either way. We have tools to warn if that happens. :)